numpy/lib/polynomial.py

```diff
@@ -441,6 +441,12 @@ def polyfit(x, y, deg, rcond=None, full=False, w=None, cov=False):
     method is recommended for new code as it is more stable numerically. See
     the documentation of the method for more information.
 
+    Do not confuse this function with the very similar
+    `numpy.polynomial.polynomial.polyfit`, which returns its coefficients in
+    increasing order of degree, and its `full` output as a list in the
```

Comment:
I feel the sentence about "full" output is hard to understand without looking at the code; I think it may be best to just remove it altogether, or maybe just '(and organises its "full" output differently)'. Basically, here we want to suggest to please look at the other code.